src/components/NewGalaxyModal.js

```diff
@@ -306,17 +316,72 @@ export const NewGalaxyModal = _.flow(
                 onlyInteger: true,
                 value: dataDiskSize,
                 style: { marginTop: '0.5rem', width: '5rem' },
+                disabled: !!currentDataDisk,
+                tooltip: !!currentDataDisk ?
+                  ['Persistent disk of size ', currentDataDisk.size, ' GB already exists and will be attached upon Galaxy creation'] :
+                  undefined,
```

Comment:
Is this portion applicable? The `div` these lines are part of, will only be rendered if `currentDataDisk` is `falsy`, in which case `!!currentDataDisk` should always be `false`, and so should `disabled` and `tooltip`?

Reply:
meant to remove this actually - will commit that with the rest of the changes from the feedback received so far. this was my original approach to not letting the user choose a disk size if they have an existing PD that is going to be attached to their app but after syncing with joy, we decided to go with a different approach seen here: 
<img width="1182" alt="Screen Shot 2021-05-12 at 10 42 48 PM" src="https://user-images.githubusercontent.com/23626109/118070159-63171480-b373-11eb-8611-660441d11082.png">